src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -1159,13 +1159,13 @@ internal static DateTime UnixSecondsToDateTime(long seconds)
             return DateTimeOffset.FromUnixTimeSeconds(seconds).DateTime;
 #else
             const int DaysPerYear = 365;
-            const int DaysPer4Years = DaysPerYear * 4 + 1;
-            const int DaysPer100Years = DaysPer4Years * 25 - 1;
-            const int DaysPer400Years = DaysPer100Years * 4 + 1;
-            const int DaysTo1970 = DaysPer400Years * 4 + DaysPer100Years * 3 + DaysPer4Years * 17 + DaysPerYear;
+            const int DaysPer4Years = (DaysPerYear * 4) + 1;
+            const int DaysPer100Years = (DaysPer4Years * 25) - 1;
+            const int DaysPer400Years = (DaysPer100Years * 4) + 1;
+            const int DaysTo1970 = (DaysPer400Years * 4) + (DaysPer100Years * 3) + (DaysPer4Years * 17) + DaysPerYear;
```

Comment:
Less readability.

Reply:
For most people it is obvious that multiplication has a higher precedence than addition, so the parenthesis are probably not necessary. However while readability might not be any better, it is not really any worse.

Reply:
Multiplicative > Additive